library/core/src/ops/drop.rs

```diff
@@ -136,6 +136,7 @@
 /// [`Copy`]: ../../std/marker/trait.Copy.html
 #[lang = "drop"]
 #[stable(feature = "rust1", since = "1.0.0")]
+#[rustc_diagnostic_item = "Drop"]
```

Comment:
`Drop` is a lang item, which already makes it known to the compiler, so it doesn't need to be a diagnostic item as well.

Reply:
That's fixed.